src/test/java/com/simplythewest/utd/ToDoItemRepositoryTest.java

```diff
@@ -16,12 +17,12 @@
 
 class ToDoItemRepositoryTest {
 
-    @Autowired
-    ToDoItemRepository todorepo1;
+    private ToDoItemRepository todorepo1;
 
     @Test
-    void createThenSave()
+    void createThenSave(ToDoItemRepository newRepo)
```

Comment:
uh, what? this isn't how you use parameterized tests in junit 5, even if it was you shouldn't do it with a repository.

Reply:
fixed, but now it says 'todorepo1' is never assigned.